### DISCUSSION TITLE
Welcome to curl Discussions!

### DISCUSSION THREAD

**Discussion by bagder** [curl/curl#6290] [General]
## 👋 Welcome!
  We’re **testing** GitHub Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Welcome to curl!

**Comment by centminmod** [comment]
Nice definitely good for projects which don't already have their own discussion forum (besides mailing lists) 

thanks @bagder 

```
curl-http3 -V
curl 7.74.0-DEV (x86_64-pc-linux-gnu) libcurl/7.74.0-DEV BoringSSL zlib/1.2.11 brotli/1.0.7 libidn2/2.0.5 libpsl/0.20.2 (+libidn2/2.0.5) libssh2/1.8.0 nghttp2/1.36.0 quiche/0.6.0
Release-Date: [unreleased]
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps mqtt pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: alt-svc AsynchDNS brotli HTTP2 HTTP3 HTTPS-proxy IDN IPv6 Largefile libz NTLM NTLM_WB PSL SSL UnixSockets
```

**Comment by kiliczsh** [comment]


**Comment by KaniketCode95** [comment]
Hi 
Im trying to stablish a connection between my iot device and local host server using curl
localhost: 127.0.0.1:portnumber

I'm using this command
curl -X POST --data-binary @kk.json http://172.30.53.xx:pornumber/

but its not reflecting any thing 
even i tried through wsl no good luck 
but through virtual box and window command prompt 
its working fine.

not sure why?
any lead will help me out.!

**Reply by bagder** [reply]
1. You should create a new discussion post/thread rather than replying here in the intro.
2. You don't want `-X POST` with `--data-binary`
3. It's not clear what you do or what you want. What happens *exactly* when you invoke the command you showed?
